assets/js/googlesitekit/datastore/user/feature-tours.js

```diff
@@ -80,19 +90,69 @@ const baseActions = {
 		invariant( slug, 'A tour slug is required to dismiss a tour.' );
 
 		return ( function* () {
+			const { select } = yield getRegistry();
+			if ( select( STORE_NAME ).isFetchingDismissTour( slug ) ) {
+				const response = select( STORE_NAME ).getDismissedTours();
+				return { response, error: undefined };
+			}
 			// Dismiss the given tour immediately.
 			yield {
 				payload: { slug },
 				type: DISMISS_TOUR,
 			};
-			// Dispatch a request to persist the dismissal.
-			const { response, error } = yield fetchDismissTour( slug );
-			if ( ! error ) {
-				yield receiveGetDismissedTours( response );
-			}
-			return { response, error };
+			// Dispatch a request to persist and receive updated dismissed tours.
+			return yield fetchDismissTour( slug );
 		}() );
 	},
+
+	receiveReadyTours( readyTours, { viewContext } = {} ) {
```

Comment:
Let's rename this alongside the related selector, to `receiveGetCurrentFeatureTours`.

Reply:
How about `receiveCurrentFeatureTours`? `receiveGetCurrentFeatureTours` sounds like it is a fetch store action.